doc/changes/latest.inc

```diff
@@ -44,6 +44,7 @@ Enhancements
 - Add example of how to obtain time-frequency decomposition using narrow bandpass Hilbert transforms to :ref:`ex-tfr-comparison` (:gh:`11116` by `Alex Rockhill`_)
 - Add ``==`` and ``!=`` comparison between `mne.Projection` objects (:gh:`11147` by `Mathieu Scheltienne`_)
 - Add ``encoding`` parameter to :func:`mne.io.read_raw_edf` and :func:`mne.io.read_raw_bdf` to support custom (non-UTF8) annotation channel encodings (:gh:`11154` by `Clemens Brunner`_)
+- Add config option ``MNE_REPR_HTML`` to disable HTML repr in notebook environments (:gh:`11159` by `Clemens Brunner`_)
```

Comment:
looking at the doc and thinking where this information could be added I found

https://mne.tools/stable/auto_tutorials/intro/50_configure_mne.html

and this tells me that maybe using mne config mechanism would be the expect thing to do...

what do you think @cbrnr @drammock ?

Reply:
I'm not sure I understand. Do you want to describe the new config option in the docs? We haven't done this for the other existing options though.